docs/account/emails.md

```diff
@@ -2,6 +2,13 @@
 
 To manage the email addresses associated with your account and your email notifications, click on your avatar on the top right-hand corner and open **Your account**, page **Emails**.
 
+## Updating your email addresses
+
+Codacy pulls email addresses from the Git provider associated with your current session. To update the email addresses displayed on Codacy, update them on your Git provider, then log out and back in to Codacy.
+
+!!! tip
+    If the updates are still not reflected on Codacy, navigate to the [access management](https://app.codacy.com/account/access-management) page, revoke the relevant Git provider or Google integration, then log out and back in to Codacy using the same provider.
```

Comment:
I suggest using a note here because the way the previous page was written makes it seem like it is common that the updates aren't immediately synced. A tip is more appropriate for something that is optional or provides an alternative (more efficient) way of doing something. Here, I'm thinking that these "troubleshooting" steps may actually be more mandatory than not. :thinking: 

```suggestion
!!! note
    If the updates are still not reflected on Codacy, navigate to the [access management](https://app.codacy.com/account/access-management) page, revoke the relevant Git provider or Google integration, then log out and back in to Codacy using the same provider.
```